### PR TITLE
PAINT-187 Espresso tests for ButtonTopLayers_RTL_LayoutTest

### PR DESCRIPTION
**Pull request by ndrnour**
Espresso Test

### CODE REVIEW

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package org.catrobat.paintroid.test.espresso.rtl;
```
Comment (cbenkovic) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java]: see comments in your previous pull request [PR-421](https://github.com/Catrobat/Paintroid/pull/421)
Here you did the reseting of the test language in a nice way with the tearDown() method, you could do the same for [PR-421](https://github.com/Catrobat/Paintroid/pull/421)

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    @Before
+    public void setUp() throws Exception {
+     launchActivityRule.getActivity();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        SharedPreferences sharedPreferences = InstrumentationRegistry.getTargetContext().getSharedPreferences("For_language", Context.MODE_PRIVATE);
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        editor.putString(LANGUAGE_TAG_KEY, "en");
```
Comment (cbenkovic) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java]: Would it better to just remove the language setting? -> editor.remove(LANGUAGE_KEY)
The default device language of our test devices is usually english

Comment (ndrnour) [reply Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java]: you are right, would be better as you wrote

**Comment by ndrnour** [comment]
@cbenkovic  please have a look again.

**Comment by ndrnour** [comment]
@cbenkovic could you please check it again & thanks.

**Pull request merged**